pkgs/applications/networking/instant-messengers/linphone/default.nix

```diff
@@ -18,6 +18,12 @@ stdenv.mkDerivation rec {
     sha256 = "0az2ywrpx11sqfb4s4r2v726avcjf4k15bvrqj7xvhz7hdndmh0j";
   };
 
+  cmakeFlags = "-DENABLE_GTK_UI=ON";
```

Comment:
Its better do use lists for `*Flags` (except some very specific cases)

Reply:
done.